sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -783,7 +784,7 @@ abstract class DDLSuite extends QueryTest with DDLSuiteBase {
         sql("ALTER TABLE tab1 RENAME TO default.tab2")
       }
       assert(e.getMessage.contains(
-        s"RENAME TEMPORARY VIEW from '`tab1`' to '`$SESSION_CATALOG_NAME`.`default`.`tab2`': " +
+        s"RENAME TEMPORARY VIEW from '`tab1`' to '`default`.`tab2`': " +
```

Comment:
This reverts back to the behavior before https://github.com/apache/spark/pull/37021 : we shouldn't blindly attach catalog names in every v1 command. In this case we shouldn't mention `SESSION_CATALOG_NAME` because temp views do not belong to any catalog.